ansible/configs/ent-demo-ate-vmc-sandbox/default_vars.yml

```diff
@@ -0,0 +1,38 @@
+---
+#### Things you can change ####
+
+# Why is this config being deployed? Override for specific environments
+# Some valid: development, ilt, production, event
+purpose: development
+
+# The {{ guid }} is used everywhere and it is what differentiates otherwise
+# identical environments. Make this unique. Usually they are 4 characters, but
+# it can be any reasonable length.
+guid: notset
+
+# These values will be added to the account tracking database
+# and may be used as resources tags as well
+owner_name: "{{ student_name }}"
```

Comment:
Best to not use `student_name` as the owner is often not a student. This is particularly confusing with workshops where the "student" ends up being the lab facilitator.